lib/ansible/module_utils/basic.py

```diff
@@ -1303,10 +1288,7 @@ def log(self, msg, log_args=None):
                 # TODO: surrogateescape is a danger here on Py3
                 journal_msg = remove_values(msg, self.no_log_values)
 
-            if PY3:
-                syslog_msg = journal_msg
-            else:
-                syslog_msg = journal_msg.encode('utf-8', 'replace')
+            syslog_msg = journal_msg
```

Comment:
This variable is only passed into two places below. I think, it could be removed and `journal_msg` passed directly.